tutorials/sim/plot_SimulatePeriodic.py

```diff
@@ -126,65 +136,169 @@
 # oscillations with this property.
 #
-# You can simulate bursty oscillations with :func:`~.sim_bursty_oscillation`.
+# Bursty oscillations can be simulated with :func:`~.sim_bursty_oscillation`.
+#
+# Burst Probability
+# ~~~~~~~~~~~~~~~~~
 #
-# To control the bursty-ness of the simulated signal, you can control the probability
-# that a burst will start or stop with each new cycle.
+# One way to control the bursty-ness of the simulated signal, is to control
+# the probability that a burst will start or stop with each new cycle.
 #
 
 ###################################################################################################
 
-# Settings for simulation
-osc_freq = 30
+# Simulation settings
 n_seconds = 3
-enter_burst = .1
-leave_burst = .1
+times = create_times(n_seconds, fs)
+
+# Define oscillation frequency
+osc_freq = 30
+
+# Burst settings
+enter_burst = 0.1
+leave_burst = 0.1
+
+###################################################################################################
 
 # Simulate a bursty oscillation
-osc = sim_bursty_oscillation(n_seconds, fs, osc_freq,
-                             enter_burst=enter_burst,
-                             leave_burst=leave_burst)
-times = create_times(n_seconds, fs)
+burst = sim_bursty_oscillation(n_seconds, fs, osc_freq,
+                               enter_burst=enter_burst,
+                               leave_burst=leave_burst)
 
 ###################################################################################################
 
-# Plot the simulated data, in the time domain
-plot_time_series(times, osc, xlim=[0, n_seconds])
+# Plot the simulated burst signal
+plot_time_series(times, burst, xlim=[0, n_seconds])
+
+###################################################################################################
+#
+# By updating the burst settings, we can change the overall probability of bursting.
+#
+# For example, we can shorten burst duration by increasing the probability to leave bursts.
+#
+# Alternatively, we can increase the number of bursts by increasing the probability
+# to enter a burst.
+#
+
+###################################################################################################
+
+# Simulate a bursty oscillation, with a higher probability to leave bursts
+short_burst = sim_bursty_oscillation(n_seconds, fs, osc_freq,
+                                     enter_burst=0.1, leave_burst=0.4)
+
+# Simulate a bursty oscillation, with a higher probability of entering bursts
+more_bursts = sim_bursty_oscillation(n_seconds, fs, osc_freq,
+                                     enter_burst=0.4, leave_burst=0.1)
 
 ###################################################################################################
+
+# Plot the simulated burst signals
+plot_time_series(times, short_burst, xlim=[0, n_seconds], title='Shorter Burst')
+plot_time_series(times, more_bursts, xlim=[0, n_seconds], title='More Bursts')
```

Comment:
These double plots render a bit weird in the same cell. Maybe separate into different cells or use custom axes.